### PR TITLE
Add optional pv viewer to limit transfer speed

### PR DESCRIPTION
**Pull request by op**
Emulates the feel for the output a bit. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ brew install coreutils
 
 #### Optional
 * [fd](https://github.com/sharkdp/fd) a modern `find` replacement, it will use this preferentially if it's installed otherwise fallback to `find`
+* [pv](https://www.ivarch.com/programs/pv.shtml) a pipe viewer with which we can
+  limit the output speed to emulate the speed back in the days. :)
 
+  Set `ANSI_ART_RATE_LIMIT` to eg `8k` to limit data rate to 8192 bps.
```
Comment (yuhonas) [inline README.md]: Nice, though can we set this to use the `ANSI_MOTD` ENV namespace to keep it consistent with the others eg. `ANSI_MOTD_RATE_LIMIT`

File: zsh-ansimotd.plugin.zsh [hunk 2]
```diff
@@ -52,14 +52,20 @@ function ansi_art_random_file {
 function ansi_art_random {
   ansi_filename="$(ansi_art_random_file)"
 
+  # optional use pv to limit rate we output
+  viewer=(cat)
+  if (( $+commands[pv] )); then
```
Comment (yuhonas) [inline zsh-ansimotd.plugin.zsh:57]: Can we change this to opt in, given some people may have `pv` installed but may not want `pv` buffered output, although cool as hell i'm not sure on the consensus if it's an expected default/always on (when pv is installed) at this point so i'd like err on the side of user choice

In the condition let's add another ENV which will need to be satisfied to enable this output perhaps something like `ANSI_MOTD_RATE_LIMIT_OUTPUT`, suggestions welcome on alternative naming!

**Review by yuhonas — CHANGES_REQUESTED** [review]
This is truly awesome! really does give those old skool BBS vibes, just some small feedback to address, nice work! 💯👏🏼  

ansi

File: zsh-ansimotd.plugin.zsh (hunk 2 above)
Comment (op) [reply zsh-ansimotd.plugin.zsh:57]: Yeah, totally agree. :+1: Updated to use the suggested name and defaulted the rate to 1tbps. ;)

**Comment by yuhonas** [comment]
LGTM 👍🏼

**Pull request merged**